packages/components/src/test/properties/NumericPropertyEditor.test.tsx

```diff
@@ -38,7 +38,7 @@ describe("<NumericPropertyEditorBase />", () => {
     await waitFor(() => expect(getByTestId("numeric-input")).to.not.be.null);
   });
 
-  it("invokes `onCommit` when input changes", async () => {
+  it("Does not invoke `onCommit` when input changes but blur event is not triggered", async () => {
```

Comment:
I don't think this test is necessary anymore (see my other comment regarding unnecessary tests).

To make sure the fix is properly tested we could improve the test below by making the user type more than once before blur. Previously that would've caused as many commits as there were `type` calls. Now it doesn't matter and commit should only happen once - on blur.